src/forms.py

```diff
@@ -0,0 +1,14 @@
+"""forum123's forms module."""
+
+
```

Comment:
Let's have one empty line between docstring and imports everywhere. See [comment](https://github.com/EugeneTurkin/forum123/pull/14/files?diff=split&w=0#r1064040645)